CHANGELOG.md

```diff
@@ -22,13 +22,16 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
   - Can be disabled in Appearance->Interface
   - Shows binding name when scoreboard is opened
 - Added option to render rotated text on screen (by @TimGoll)
+- Add the required string translations (by @sbzlzh)
+- Improved translation of some Simplified Chinese strings (by @TheOnly8Z)
```

Comment:
Can you move those below to the changed section, where you already added an entry?
Also rephrase the first point to something like: `Added all new translation strings`

Reply:
ok

Reply:
So I didn't make a mistake in modifying it, did I!